aa.py

```diff
@@ -288,57 +288,70 @@ def ResetBash():
 #####AA Zone#####
 '''
 
-
+def DetectShell():
```

Comment:
let's check if its none as well, and return bash and zsh rather than .bashrc and .zshrc